test/test_backend/routes/tst_graph.py

```diff
@@ -32,70 +38,100 @@
-                         4270049, 4044391, 4044392, 4129519, 4129524, 4129525, 35625724], [4252356, 44833365], {}]
-        }
-    }
-}
 
-# todo: at end, revert class, method, and file name from tst -> test
 # class TestGraph(unittest.TestCase):
 class TstGraph:
-
-    async def tst_get_connected_subgraph(self):
+    """Tests for graph.py"""
+    test_cases: List[Dict] = []
+
+    # noinspection PyPep8Naming
+    @classmethod
+    def setUpClass(cls):
+        """Set up"""
+        csv_file = StringIO(TEST_CASES_FAST.strip())
+        cases: List[Dict] = list(csv.DictReader(csv_file, delimiter=',', quotechar='"'))
+        for case in cases:
+            for k, v in case.items():
+                if k == 'codeset_ids':
+                    case[k] = v.split(',')
+        cls.test_cases = cases
+
+    async def _create_inputs__get_connected_subgraph(self):
         """Test indented_concept_list()"""
-        # todo; enable test cases from CSV
-        for case in test_cases_list:
-            codeset_ids = case['codeset_ids'].split(',')
-            sg, nodes_in_graph, orphans_not_in_graph, hidden = (
-                get_connected_subgraph(REL_GRAPH, codeset_ids, extra_concept_ids, hide_vocabs))
-            # todo: assertions
-            j = json.dumps(
-                [list(sg.edges), list(nodes_in_graph), list(orphans_not_in_graph), hidden])
-            print()
-
-
+        self.setUpClass()  # temp until tests finished
+        for test_type, test_name, codeset_ids, timeout_secs in [x.values() for x in self.test_cases]:
+            # TODO: temp
+            if test_name not in ['antibiotics 1']:
+                continue
+            input_path = THIS_INPUT_DIR / CONNECTED_SUBGRAPH_FILENAME_PATTERN.format(test_name)
+            sg, nodes_in_graph, preferred_concept_ids, orphans_not_in_graph, hidden = \
+                get_connected_subgraph(REL_GRAPH, codeset_ids, EXTRA_CONCEPT_IDS, HIDE_VOCABS)
+            #
+            normalized_hidden = {k: sorted(list(v)) for k, v in hidden.items()}
+            results = {
+                'sg.edges': sorted(list(sg.edges)),
+                'nodes_in_graph': sorted(list(nodes_in_graph)),
+                'preferred_concept_ids': sorted(list(preferred_concept_ids)),
+                'orphans_not_in_graph': sorted(list(orphans_not_in_graph)),
+                'hidden': normalized_hidden,
+            }
+            with open(input_path, 'w') as f:
+                f.write(json.dumps(results, indent=4))
+
+    async def tst_get_connected_subgraph(self, only_fast_cases=True):
+        """Test indented_concept_list()"""
+        self.setUpClass()  # temp until tests finished
+        for test_type, test_name, codeset_ids, timeout_secs in [x.values() for x in self.test_cases]:
+            # Do just 1 fast test case, or all test cases?
+            if only_fast_cases and test_name not in ['many-small', 'single-small', 'single-small-second-time']:
```

Comment:
## `only_fast_cases`
Some of the tests were taking very long, especially `'antibiotics 1'`, which I think was taking over 30 minutes. So, I decided for now to just test `['many-small', 'single-small', 'single-small-second-time']`. These tests can complete in <20 seconds. I added a param called `only_fast_cases` and defaulted to `True` for this behavior.